### PR TITLE
exp save: always dvc commit changes

### PR DESCRIPTION
**Pull request by dberenbaum**
See https://github.com/iterative/dvclive/issues/408

### CODE REVIEW

File: dvc/repo/experiments/save.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def save(
     executor = queue.init_executor(repo.experiments, entry)
 
     save_result = executor.save(
-        executor.info, force=force, include_untracked=include_untracked
+        executor.info, force=True, include_untracked=include_untracked
```
Comment (daavoo) [inline dvc/repo/experiments/save.py:39]: 👍 good to decouple from the other `force`

**Review by daavoo — APPROVED** [review]

**Review by daavoo — CHANGES_REQUESTED** [review]
2 different tests are failing in 2 different envs. Not sure why

https://github.com/iterative/dvc/actions/runs/3694378088/jobs/6255496899#step:5:952
https://github.com/iterative/dvc/actions/runs/3694378088/jobs/6255497444#step:5:985

**Comment by dtrifiro** [comment]
I'd wait for for #8690 before taking this on

**Comment by dberenbaum** [comment]
@dtrifiro Should I close and we can discuss/incorporate into that PR?

**Comment by dtrifiro** [comment]
Yeah, that'd be best.

**Pull request closed without merging**